package.json

```diff
@@ -36,12 +36,13 @@
     "devDependencies": {
         "@babel/core": "^7.12.3",
         "@babel/preset-env": "^7.12.1",
-        "@dhis2/cli-app-scripts": "^4.0.4",
+        "@dhis2/cli-app-scripts": "^6.2.0",
         "@dhis2/cli-style": "^8.3.0",
         "@dhis2/cli-utils-docsite": "^1.3.0",
         "concurrently": "^6.2.0",
         "eslint-plugin-cypress": "^2.11.2",
         "eslint-plugin-prettier": "^3.2.0",
-        "jest": "^26.6.3"
+        "jest": "^26.6.3",
+        "wait-on": "^5.3.0"
```

Comment:
Do you want to move `wait-on` and `concurrently` into the example/testing apps in this PR or in another one?

Reply:
It's moved in this PR, but in a monorepo the devDependencies go in the root package, not in the subpackages. I realize that it is slightly awkward for an example-app to omit some dependencies but having them here is more problematic than not having them.